src/main/java/duke/AddCommand.java

```diff
@@ -0,0 +1,54 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+
+public class AddCommand extends Command {
+    private TaskType type;
+    private String description;
+    private String time;
+
+    public AddCommand(TaskType type, String description) {
+        this.type = type;
+        this.description = description;
+        this.time = "";
+    }
+
+    public AddCommand(TaskType type, String description, String time) {
+        this.type = type;
+        this.description = description;
+        this.time = time;
+    }
+
```

Comment:
Perhaps you can have a Javadoc comment here?

Reply:
Alright thanks